src/main/kotlin/org/elm/ide/test/run/ElmTestLocator.kt

```diff
@@ -32,7 +34,9 @@ object ElmTestLocator : FileUrlProvider() {
             }
 
             DESCRIBE_PROTOCOL, TEST_PROTOCOL -> {
-                val (filePath, labels) = LabelUtils.fromLocationUrlPath(path)
+                // TODO [tests-folder]: replace hard-coded string with constant defined in `ElmProject` when that change
+                //  is merged in.
+                val (filePath, labels) = LabelUtils.fromLocationUrlPath(path, metainfo ?: "tests")
```

Comment:
I would get rid of the fallback when `metainfo` is null. The fallback will not always be correct, so I'd rather that it just blows up. e.g. `error("missing path to tests dir")`

Reply:
Agreed. Done.